config/envs/default.py

```diff
@@ -25,5 +26,19 @@ class DefaultConfig:
         "0ddadcb3-ebe7-44f9-90e6-80ff3b61e0cb",
     )
 
+    # Magic Link Fields
+    NOTIFICATION_MAGIC_LINK_CONTACT_HELP_EMAIL = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_CONTACT_HELP_EMAIL
+    )
+    NOTIFICATION_MAGIC_LINK_MAGIC_LINK_URL = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_MAGIC_LINK_URL
+    )
+    NOTIFICATION_MAGIC_LINK_REQUEST_NEW_EMAIL_URL = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_REQUEST_NEW_EMAIL_URL
+    )
+    NOTIFICATION_MAGIC_LINK_FUND_NAME = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_FUND_NAME
+    )
```

Comment:
I may have misunderstood this so is it okay to have a quick chat in the morning ? 🙂